CD into iOS folder before running agvtool

maxstubbersfield opened the pull request:
I think for most use cases where people copy and paste this script they will forget to `cd` into the ios directory.

priitlatt commented on content/building/build-versioning.md:
````diff
@@ -103,6 +103,7 @@ Add the following script under your `scripts` field for `codemagic.yaml`, or as
 ```bash
 export APP_STORE_CONNECT_PRIVATE_KEY=$(echo $APP_STORE_CONNECT_PRIVATE_KEY | base64 --decode) # if you encrypted the file itself, not its content
 LATEST_BUILD_NUMBER=$(app-store-connect get-latest-app-store-build-number '1234567890') # The argument is your application's Apple ID
+cd $FCI_BUILD_DIR/ios
````
Since default working directory is anyway `$FCI_BUILD_DIR`, I find that just `cd ./ios` would be more appropriate as otherwise there could be some confusion on what does the used variable exactly mean.

priitlatt reviewed (commented):
A little cutout from the `agvtool` manpage:
```
Usage
  agvtool should be invoked with the working directory set to your project directory (the folder containing your .xcodeproj project file)
```
In the light of that, I'd say that the changes introduced here are indeed necessary, but perhaps the build dir prefixes are somewhat obsolete. Also consider adding comments to relevant script lines to indicate why changing directory is necessary.

maxstubbersfield commented:
> A little cutout from the `agvtool` manpage:
> 
> ```
> Usage
>   agvtool should be invoked with the working directory set to your project directory (the folder containing your .xcodeproj project file)
> ```
> 
> In the light of that, I'd say that the changes introduced here are indeed necessary, but perhaps the build dir prefixes are somewhat obsolete. Also consider adding comments to relevant script lines to indicate why changing directory is necessary.

Thanks for your review. I have addressed your comments and have updated the PR.

Cheers!

helinanever commented:
Thank you for your contribution, @maxstubbersfield !
